File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-03/plagiarized/L3/01/L3.kt
package IR

import java.util.Scanner

fun main() {
    val sc = Scanner(System.`in`)
    var berat = 0.0
    var inci = 0.0
    var tinggi = 0.0
    var bmi = 0.0
    var feet = 0.0

    print("Enter weight in pounds: ")
    berat = sc.nextDouble()

    print("Enter feet: ")
    feet = sc.nextDouble()
    print("Enter inches: ")
    inci = sc.nextDouble()
    tinggi = feet * 12 + inci

    bmi = berat * 0.45359237 / ((tinggi * 0.0254) * (tinggi * 0.0254))

    println("BMI is $bmi")
    when {
        bmi < 18.5 -> println("Underweight")
        bmi < 25 -> println("Normal")
        bmi < 30 -> println("Overweight")
        else -> println("Obese")
    }
}

File: datasets/IR-Plag/convertai-original-java-rest-python/case-03/plagiarized/L3/01/L3.py
import math

berat = float(input("Enter weight in pounds: "))
feet = float(input("Enter feet: "))
inci = float(input("Enter inches: "))
tinggi = feet * 12 + inci
bmi = berat * 0.45359237 / ((tinggi * 0.0254) ** 2)

print(f"BMI is {bmi:.2f}")
if bmi < 18.5:
    print("Underweight")
elif bmi < 25:
    print("Normal")
elif bmi < 30:
    print("Overweight")
else:
    print("Obese")

